src/js/components/covid19/AmountsVisualization.jsx

```diff
@@ -39,8 +40,10 @@ const AmountsVisualization = ({
     const [remainingBalanceRectangleData, setRemainingBalanceRectangleData] = useState(null);
     const [totalLineData, setTotalLineData] = useState(null);
     const [outlayLineData, setOutlayLineData] = useState(null);
-    const [obligationLineData, setObligationLineData] = useState(null);
-    const [remainingBalanceLineData, setRemainingBalanceLineData] = useState(null);
+    const [obligationLineDataOne, setObligationLineDataOne] = useState(null);
```

Comment:
Do these represent 4 different scenarios? Either add a comment or use a more descriptive name to improve readability. Inspecting the viz from the browser, it looks like these lines stack up to form what visually looks like a single line, so I'm not following why it's broken out into four pieces.

Reply:
The one remaining balance line has been broke up into four different lines due to the overlap captured in the mock.

https://bahdigital.invisionapp.com/share/BCIAEH6UVTR#/screens/296044453

Reply:
Makes sense - thank you! Can you leave a comment to that effect? Something like `the part of the line that can overlap with Total Remaining Balance text`, etc.